### ISSUE TITLE
Gdrive Stremio Add-on

### ISSUE THREAD

**Issue by rihaq** [Purfview/IMDb-Scout-Mod#121]
https://github.com/ssnjr2002/stremio-gdrive

Api /stream/<movie/series>/<imdb_id>.json

**Comment by Purfview** [comment]
Please explain what this means, I'm not a fortune teller, unfortunately. :)

**Comment by SonaliBendre** [comment]
example movie search -  https://torrentio.strem.fun/stream/movie/tt1745960.json

example series search - https://torrentio.strem.fun/stream/series/tt3032476:1:1.json

**Comment by Purfview** [comment]
> example movie search - https://torrentio.strem.fun/stream/movie/tt1745960.json
> 
> example series search - https://torrentio.strem.fun/stream/series/tt3032476:1:1.json

Thanks for examples, now I understand what he meant by "Api /stream/<movie/series>/<imdb_id>.json", but I still don't understand what end stuff he is asking for, what such icon/button should do.

Does "https://torrentio.strem.fun" part should be custom URL settable in the settings?
Those links can be used to search things, but what is the URL to go to the actual content or stream [or whatever that stremio app does]?

**Comment by rihaq** [comment]
> > example movie search - https://torrentio.strem.fun/stream/movie/tt1745960.json
> > example series search - https://torrentio.strem.fun/stream/series/tt3032476:1:1.json
> 
> Thanks for examples, now I understand what he meant by "Api /stream/<movie/series>/<imdb_id>.json", but I still don't understand what 
> Does "https://torrentio.strem.fun" part should be custom URL settable in the settings? Those links can be used to search things, but what is the URL to go to the actual content or stream [or whatever that stremio app does]?

Yes yes, custom. This link is just for example, showing you  how it works.
I've my own instance. 
https://github.com/ssnjr2002/stremio-gdrive
Please add a site name "stremio-gdrive" on your userscript
